script/run-docker-nightwatch.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-node src/platform/testing/e2e/test-server.js --buildtype=vagovprod --host=0.0.0.0 --port=3001 &
+node src/platform/testing/e2e/test-server.js --buildtype=vagovprod &
```

Comment:
There's a lot of discrepancy between port numbers and that data is passed all over our code base. By removing this, E2E tests will execute consistently on port `3333`.